### PR TITLE
wrap interactor's actions to show them in storybook interactions debugger

### PR DESCRIPTION
**Pull request by wKich**
## Motivation

Storybook announced new feature [interactive stories](https://storybook.js.org/blog/interactive-stories-beta/).

## Approach

This spike shows how we can use that feature with our interactors.

- [ ] The `track` method isn't exported for public usage.
- [ ] We don't have a way to get interactor's and action's names as well as action's arguments to properly show them in Storybook debugger's UI.
- [ ] We show all nested actions instead of top-level one.

### CODE REVIEW

**Comment by cowboyd** [comment]
Dropping this here to track changes to storybook required to implement this #16667

File: packages/globals/src/globals.ts [hunk 1]
```diff
@@ -80,7 +99,12 @@ export function setInteractorTimeout(ms: number): void {
...
 
 export function addActionWrapper<T>(
-  wrapper: (description: string, action: () => Promise<T>, type: ActionType) => () => Promise<T>
+  wrapper: (
+    description: string,
+    action: () => Promise<T>,
+    type: ActionType,
+    options: InteractorOptions
+  ) => () => Promise<T>
 ): () => boolean {
```
Comment (cowboyd) [inline packages/globals/src/globals.ts:108]: It looks like this wrapper should be a type:

```ts
export interface InteractionEvent<T> {
  description: string;
  action: () => Promise<T> ;
  type: ActionType;
  options: InteractionOptions;
}

export type InteractionHook<T> = (event: InteractionEvent<T>) => Promise<T>;

export function addActionWrapper<T>(hook: InteractionHook<T>): () => boolean {
  // add it
}
```

Comment (wKich) [reply packages/globals/src/globals.ts]: Done. That was a little bit tricky to not break things

**Comment by wKich** [comment]
Closed in favor of #162, #163, #164

**Pull request closed without merging**